modules/imgproc/src/sumpixels.cpp

```diff
@@ -10,7 +10,7 @@
 //                           License Agreement
 //                For Open Source Computer Vision Library
 //
-// Copyright (C) 2000-2008, Intel Corporation, all rights reserved.
+// Copyright (C) 2000-2008,2019 Intel Corporation, all rights reserved.
```

Comment:
It is my understanding that this is not legally allowed.  You cannot back-date copyright and since I did not personally publish anything in the range of 2008-2018, it is inaccurate for me to specify any year in that range.  I was told this by our legal team a number of years ago when I was training our new hires on coding practices and had to have a section on copyright.  For this reason, I believe I need to use a comma to specify my addition of content in 2019 only.